### PR TITLE
Set up TypeScript, add README

### PR DESCRIPTION
**Pull request by RobbinHabermehl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+# Contents
+
+- [Description](#description)
+- [Installation](#installation)
+- [Usage](#usage)
+
+## Description
+
+When you're working on a project with multiple people you might've noticed that everyone is using different script names for the most common tasks. Where one might use `npm start` for starting the development mode of an application, someone else is using that command for starting the production build. Especially when you're working on more than one application you might have a hard time remembering all the right commands.
```
Comment (RubenWerdmuller) [inline README.md]: Aren't multiple people in the same project forced to use the same script names?

```suggestion
When you're working on multiple applications with others you might have noticed that more often than not different script names are used for the most common tasks. Where one might use `npm start` for starting the development mode of an application, someone else is using that command for starting the production build.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+   ...
+   ```
+
+## Usage
+
+Execute `npm start`, that's all!
+
+## Credits
+
+Created with ♥ by [Humanoids](https://humanoids.nl)
```
Comment (RubenWerdmuller) [inline README.md:71]: ♥

**Review by RubenWerdmuller — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (RobbinHabermehl) [reply README.md]: I'd try to come up with an example where it's most likely to encounter mismatches. So I though of a project you're working on together, then it's very common the standards of each regarding script names differ. Working on different applications that are all your own won't cause that situation 😅

File: README.md (hunk 2 above)
Comment (RobbinHabermehl) [reply README.md:71]: 🤖

File: README.md (hunk 1 above)
Comment (RubenWerdmuller) [reply README.md]: Ah! I meant multiple applications with multiple people. Edited the suggestion. I interpret the current version as multiple people working on one application with one set of scripts

Comment (RobbinHabermehl) [reply README.md]: I've combined the two now, thanks!

**Pull request merged**